pkg/cli.go

```diff
@@ -52,63 +56,70 @@ func NewApp() *cli.App {
-				cli.ShowAppHelp(c)
-				os.Exit(2)
-			}
+			{
+				Name:      "create",
+				Usage:     "Create a new project with the specified supporeted template",
+				UsageText: "jumpstart create -t={supported template name} [project-name]",
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:    "template",
+						Aliases: []string{"t"},
+						Usage:   "template to use",
+					},
+				},
+				Action: func(c *cli.Context) error {
+					// Check if default template directory exists
+					_, err := os.Stat(absPath(DEFAULT_TEMPLATES_DIR))
+					if os.IsNotExist(err) {
+						fmt.Printf("Please run `jumpstart template sync` to seed templates\n\n")
+						os.Exit(2)
+					}
 
-			dm, err := NewDerivedMetadata(c.Context, c.Args().First())
-			if err != nil {
-				return err
-			}
-			logrus.Infof("derived metadata: %+v", *dm)
+					tid := c.String("template")
+					if tid == "" {
+						fmt.Printf("-t | --template is required\n\n")
+						cli.ShowAppHelp(c)
+						os.Exit(2)
+					}
 
-			sshKeys := NewSSHKeys()
-			{
-				count := len(sshKeys.List())
-				if count == 0 {
-					fmt.Println("No SSH keys found! Please generate one and try again!")
-					fmt.Println("See https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent for more information")
-					os.Exit(2)
-				} else if count > 1 {
-					fmt.Printf("Multiple SSH keys found! Please select one to use:\n\n")
-					for i, k := range sshKeys.List() {
-						fmt.Printf("%d. %s\n", i+1, k)
+					dm, err := NewDerivedMetadata(c.Context, c.Args().First())
+					if err != nil {
+						return err
 					}
-					fmt.Printf("\nEnter selection: ")
+					logrus.Infof("derived metadata: %+v", *dm)
 
-					// Read selection from user
-					var selection int
-					fmt.Scanln(&selection)
-					sshKeys.SetSelected(selection - 1)
-				}
-			}
-			fmt.Printf("\nUsing ssh key: %s\n", sshKeys.Get())
+					sshKeys := NewSSHKeys()
+					{
+						count := len(sshKeys.List())
+						if count == 0 {
+							fmt.Println("No SSH keys found! Please generate one and try again!")
+							fmt.Println("See https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent for more information")
+							os.Exit(2)
+						} else if count > 1 {
+							fmt.Printf("Multiple SSH keys found! Please select one to use:\n\n")
+							for i, k := range sshKeys.List() {
+								fmt.Printf("%d. %s\n", i+1, k)
+							}
+							fmt.Printf("\nEnter selection: ")
 
-			gitSSHKeys, err := ssh.NewPublicKeysFromFile("git", sshKeys.Get(), "")
-			if err != nil {
-				return err
-			}
+							// Read selection from user
+							var selection int
+							fmt.Scanln(&selection)
+							sshKeys.SetSelected(selection - 1)
+						}
+					}
+					fmt.Printf("\nUsing ssh key: %s\n", sshKeys.Get())
+					fmt.Print("Enter ssh passphrase if you added it while ssh-key generation: ")
+					var passphrase string
```

Comment:
we should allow the passphrase to be inputed at the CLI, env_var as well as runtime.

Reply:
done